package.json

```diff
@@ -22,28 +22,27 @@
   "publishConfig": {
     "access": "public"
   },
+  "workspaces": [
+    "demo"
+  ],
   "scripts": {
-    "all": "yarn lint && yarn types && yarn test",
     "build": "tsc --project tsconfig.build.json",
     "clean": "rimraf ./build",
-    "demo": "yarn build && yarn --cwd demo storybook:axe",
-    "demo:axeOnly": "yarn build && yarn --cwd demo storybook:axeOnly",
-    "demo:build": "yarn --cwd demo build-storybook",
+    "demo": "npm run build && npm run --workspace=demo storybook:axe --",
```

Comment:
Are the trailing `--` needed or just for convenience?

Reply:
Good eye. They're needed to pass through args to the sub-scripts.

e.g., lets me run something like `npm run demo -- --headless=false` and the `--headless` option will get passed through to `storybook:axe`.

Reply:
Oh weird, I thought having the trailing `--` in the command would allow you to just pass args directly. Like `npm run demo --headless=false` but if you didn't leave the trailing `--` then you'd need to provide it yourself `npm run demo -- --headless=false`

Reply:
Ah, that might also be the case.

Here it was more about passing things right to the sub command. But it may also have other affects.